[IMP] odoo-print: Add number of copies to qweb-cpcl templates

joe-gb opened the pull request:
Adding in copies into the print statements of cpcl templates
automatically. CPCL requires the number of labels to be sent
as part of the CPCL itself - passing qty to LPR has no effect.

parisiale commented:
Tested a fedora instance where the `python3-magic` package is not installed.
Before: 2 printer unit tests failures (tests 15 and 16).
After: all passed!
👍

Commit:
[IMP] odoo-print: Add number of copies to qweb-cpcl templates

Adding in copies into the print statements of cpcl templates
automatically. CPCL requires the number of labels to be sent
as part of the CPCL itself - passing qty to LPR has no effect.
